<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_member_ui</class>
 <widget class="QDialog" name="add_member_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Member</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#CBE6C7;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="text">
    <string>ADD MEMBER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="member_email_entry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>345</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
  </widget>
  <widget class="QLabel" name="address_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Address : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="member_name_entry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>105</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="member_address_entry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>185</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Name :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="member_contact_entry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>265</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
  </widget>
  <widget class="QLabel" name="contact_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Contact : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="email_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Email Id :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="add_member_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>405</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #122C4F;
	border: 1px solid #000000;
    color: #CBE6C7;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #0D203A;
}

QPushButton:pressed {
    background-color: #091525;
}
</string>
   </property>
   <property name="text">
    <string>Add </string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>288</y>
     <width>110</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>2</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>278</y>
     <width>100</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>2</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>110</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>2</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>100</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>2</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
